site/content/docs/5.2/components/badge.md

```diff
@@ -43,6 +53,13 @@ Unless the context is clear (as with the "Notifications" example, where it is un
 
 Use utilities to modify a `.badge` and position it in the corner of a link with an icon.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This component variant should be used **only** inside a header component. Please refer to our Boosted [Navbars]({{< docsref "/examples/navbars" >}}) examples.
+
+For other usages, consider using our Boosted [Tags]({{< docsref "/components/tags" >}}) component. You can also refer to the [Tags](https://system.design.orange.com/0c1af118d/p/975c09-tags/b/24dde8) guidelines on the Orange Design System website.
```

Comment:
This sentence should not be used here. We're not going to use tags when positioned. Maybe this sentence should be moved to the first callout of this page; with Headings for example, or deleted.

Reply:
OK for deleted this sentence. IMO not to add to the callout of #Headings because the usage for our Tags is not the same one as the ones presented in #Headings.
=> What about this new version in #Positioned ?
> This component variant should be used **only** inside a header component. 
>
> Please refer to our Boosted [Navbars]({{< docsref "/examples/navbars" >}}) examples. You can also refer to the [Global headers](https://system.design.orange.com/0c1af118d/p/37609b-global-headers/b/366c91) guidelines on the Orange Design System website.